node/src/protocol/mod.rs

```diff
@@ -212,27 +213,14 @@ impl MpcSignProtocol {
             .with_label_values(&[&my_account_id])
             .set(1);
         let mut queue = MpcMessageQueue::default();
+        let mut last_state_update = Instant::now();
         loop {
             tracing::debug!("trying to advance mpc recovery protocol");
-            let contract_state = match rpc_client::fetch_mpc_contract_state(
-                &self.ctx.rpc_client,
-                &self.ctx.mpc_contract_id,
-            )
-            .await
-            {
-                Ok(contract_state) => contract_state,
-                Err(e) => {
-                    tracing::error!("could not fetch contract's state: {e}");
-                    tokio::time::sleep(Duration::from_secs(1)).await;
-                    continue;
-                }
-            };
-            tracing::debug!(?contract_state);
             loop {
                 let msg_result = self.receiver.try_recv();
                 match msg_result {
                     Ok(msg) => {
-                        tracing::debug!("received a new message");
+                        tracing::debug!(?msg, "received a new message");
```

Comment:
this log here likely spams our logs. Do you think we need? It's not like we need to see what type of message we're getting though